src/plugins/data/public/ui/query_string_input/_query_bar.scss

```diff
@@ -10,12 +10,18 @@
  */
 
 .osdQueryBar__wrap {
-  max-width: 100%;
-  z-index: $euiZContentMenu;
-
   .euiToolTipAnchor {
     height: 100%;
   }
+
+  &.euiFormControlLayout--group.euiFormControlLayout--compressed {
+    overflow: initial;
+  }
```

Comment:
is this from something that wasn't backported?

Reply:
yes, this PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8343, do we want to do backports in order?